Makefile

```diff
@@ -76,7 +76,7 @@ bench-async:
 # Vulnerability check
 
 target/nvd:
-	clojure -Xnvd check :classpath '"'"$$(clojure -Spath -A:db-h2:db-sqlite:db-postgres)"'"'
+	clojure -Xnvd check :classpath '"'"$$(clojure -Spath -A:db-h2:db-sqlite:db-postgres)"'"' :config-filename '"nvd_config.json"'
```

Comment:
Feel like `nvd_config.json` (and `nvd_suppression.xml`) could go in the `dev_resources` folder.

Reply:
(Unless the `:nvd` alias requires that the files be at the top level)

Reply:
sure

Reply:
https://github.com/yetanalytics/lrsql/pull/222/commits/2b7bf49f3c6c5af6cd7ab236a851cbdf4dc1475f